js/src/components/SettingsReplacementVariableEditors.js

```diff
@@ -75,7 +75,7 @@ class SettingsReplacementVariableEditors extends React.Component {
 				reactReplacevarPageType,
 			} = targetElement.dataset;
 			return ReactDOM.createPortal(
-				<SettingsTitleReplacementVariableEditor
+				<SettingsReplacementVariableField
```

Comment:
You added a required fieldId to settingsReplacementVariableField, but you never added fieldId To this usage of it. Same with settingsReplacementVariableEditor.

Reply:
It's there, I just did not edit it 👼